src/pages/webinar/update/index.tsx

```diff
@@ -167,6 +167,7 @@ const UpdateWebinar: React.FC = () => {
 
 				if (webinarId) {
 					dispatch(updatedWebinar(webinarId, webinarFormData));
+					location.reload();
```

Comment:
Can't we update state of the image to null

Reply:
I tried to do it in some ways. But finally, this was fast and possible to continue with the requirement.